jfr-connection/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry JFR Connection"
+otelJava.moduleName.set("io.opentelemetry.contrib.jfr.connection")
+
+dependencies {
+  testImplementation("org.junit.jupiter:junit-jupiter:5.9.2")
```

Comment:
I believe this is also brought by `otel.java-conventions`
```suggestion
```

Reply:
commit 388acaf